README.md

```diff
@@ -54,6 +58,90 @@ jobs:
           push: true
 ```
 
+### Git context
+
+By default, this action will use the local bake definition, so you need to
+use the [`actions/checkout`](https://github.com/actions/checkout/) action to
+check out the repository.
+
+If you want to use a remote bake definition, you can use the `source` input
+to specify a remote bake definition to use. Default Git context can be provided
+using the [Handlebars template](https://handlebarsjs.com/guide/) expression
+`{{defaultContext}}`:
+
+```yaml
+name: ci
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  bake:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/bake-action@v4
+        with:
+          source: "{{defaultContext}}"
```

Comment:
Really confusing that this on has quotes and no `$` and 5 line up it is completely different.

Reply:
Yes we are using [Handlebars template](https://handlebarsjs.com/guide/) which are not the same as GHA ones.

Reply:
Why not just fill it in with the env variable?

Reply:
We would need a previous step to fill in this env var so it could be used in this input.

Reply:
I'm still confused by this syntax. Are there more examples of this being something users are used to? If kept then is `defaultContext` the best name or should it be more like `GITHUB_REF` to match https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables ?

How will this look like after we switch the opt-in? When users want to make sure the behavior does not change with next release they should define `source: . ` atm? And they can also use a Git/HTTP URL in this field?

Reply:
I changed the syntax to the raw one in the README if they want to try out the Git context in their workflow. Didn't to bring another magic string in `source` as we are going to opt-out Git context in next major release anyway.